### PR TITLE
T-32182 Migrate rubicon trade

### PR DESCRIPTION
**Pull request by couralex6**

### CODE REVIEW

File: models/rubicon/arbitrum/rubicon_arbitrum_trades_legacy.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{ config( 
+  schema = 'rubicon_arbitrum',
+  alias = alias('table_name', legacy_model=True),
```
Comment (jeff-dude) [inline models/rubicon/arbitrum/rubicon_arbitrum_trades_legacy.sql:3]: interesting, this one didn't have a legacy file?

File: models/rubicon/optimism/rubicon_optimism_schema.yml [hunk 2]
```diff
@@ -197,7 +193,7 @@ models:
       - &bought_amount_raw
         name: bought_amount_raw
         description: "the raw amount of the token that the offer has bought"
-      - &sell_amount_usd
+      - &sell_amount_usd:
```
Comment (jeff-dude) [inline models/rubicon/optimism/rubicon_optimism_schema.yml:196]: accident to add `:` here?
i can't recall exact scenario, but this caused issues in the past and we had to remove it

File: models/rubicon/rubicon_schema.yml [hunk 3]
```diff
@@ -142,7 +139,7 @@ models:
       - &bought_amount_raw
         name: bought_amount_raw
         description: "the raw amount of the token that the offer has bought"
-      - &sell_amount_usd
+      - &sell_amount_usd:
```
Comment (jeff-dude) [inline models/rubicon/rubicon_schema.yml:142]: same here

File: models/rubicon/rubicon_trades.sql [hunk 4]
```diff
@@ -1,9 +1,9 @@
-{{ config(
+{{ config(tags=['dunesql'],
         alias = alias('trades'),
         post_hook='{{ expose_spells(\'["optimism"]\',
                                 "project",
                                 "rubicon",
-                                \'["msilb7, denver"]\') }}'
+                                \'["msilb7"]\') }}'
```
Comment (jeff-dude) [inline models/rubicon/rubicon_trades.sql:6]: accident to remove this contributor?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/rubicon/arbitrum/rubicon_arbitrum_trades_legacy.sql (hunk 1 above)
Comment (couralex6) [reply models/rubicon/arbitrum/rubicon_arbitrum_trades_legacy.sql:3]: I think it was added recently.

**Review by antonio-mendes — APPROVED** [review]
LGTM now, I've removed the `schema` part from the beginning of the models. I saw it's not there for most other models so I assume we don't actually need it @jeff-dude

**Pull request merged**